boards/pic32-clicker/Makefile.features

```diff
@@ -3,5 +3,6 @@ CPU_MODEL = p32mx470f512h
 
 # Put defined MCU peripherals here (in alphabetical order)
 FEATURES_PROVIDED += periph_gpio
+FEATURES_PROVIDED += periph_gpio_irq
```

Comment:
Those should be CPU features

Reply:
Ok, I moved that to `cpu/mips_pic32_common/Makefile.features`.